src/app_service/service/contacts/async_tasks.nim

```diff
@@ -21,6 +21,8 @@ const lookupContactTask: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
     # TODO refactor those calls to use the new backend and also do it in a signle call
     pubkey = ens_utils.pubkey(arg.value)
     address = ens_utils.address(arg.value)
+  else:
+    pubkey = ""
```

Comment:
@Khushboo-dev-cpp just because of consistency and to be sure, would be good to add `address = ""` as well in this else block.

Reply:
Sure :)